deepchem/trans/transformers.py

```diff
@@ -607,3 +607,56 @@ def transform(self, dataset):
   def untransform(self, z):
     raise NotImplementedError(
         "Cannot untransform datasets with FitTransformer.")
+
+
+class IRVTransformer():
+  """Performs transform from ECFP to IRV features(K nearest neibours)."""
+
+  def __init__(self, K, n_tasks, dataset, transform_y=False, transform_x=False):
+    """Initializes IRVTransformer.
+    Parameters:
+    ----------
+    dataset: dc.data.Dataset object
+      train_dataset
+    K: int
+      number of nearest neighbours being count
+    n_tasks: int
+      number of tasks
+
+    """
+    self.X = dataset.X
+    self.n_tasks = n_tasks
+    self.K = K
+    self.y = dataset.y
+    self.w = dataset.w
+    self.transform_x = transform_x
+    self.transform_y = transform_y
+
+  def realize(self, similarity, y, w):
```

Comment:
Could you add doc-strings for `realize()` and `X_transform()` along with a sample usage example in the docs for `IRVTransformer`? (I believe some of the other classes in `transformers.py` have usage examples in their docs)